src/pageComponents/dashboard/sidePane/sidePaneOptionsMenu.tsx

```diff
@@ -39,65 +40,30 @@ const SidePaneOptionsMenu = () => {
 		(state) => state.toggleShowSettingsModal,
 	);
 
-	const optionsMenuList = [
-		{
-			icon: <SearchIconGray />,
-			name: "Search",
-			href: `/${SEARCH_URL}`,
-			current: currentPath === SEARCH_URL,
-			id: 0,
-			count: undefined,
-			iconColor: "",
-		},
-		{
-			icon: <InboxIconGray />,
-			name: "Inbox",
-			href: `/${UNCATEGORIZED_URL}`,
-			current: currentPath === UNCATEGORIZED_URL,
-			id: 2,
-			count: bookmarksCountData?.data?.uncategorized,
-			iconColor: "",
-		},
-		{
-			icon: <HomeIconGray />,
-			name: "All",
-			href: `/${ALL_BOOKMARKS_URL}`,
-			current: currentPath === ALL_BOOKMARKS_URL,
-			id: 1,
-			count: bookmarksCountData?.data?.allBookmarks,
-			iconColor: "",
-		},
-
-		{
-			icon: <TrashIconGray />,
-			name: "Trash",
-			href: `/${TRASH_URL}`,
-			current: currentPath === TRASH_URL,
-			id: 3,
-			count: bookmarksCountData?.data?.trash,
-			iconColor: "",
-		},
-		{
-			icon: <SettingsIcon />,
-			name: "Settings",
-			href: "",
-			current: false,
-			id: 4,
-			count: undefined,
-			iconColor: "",
-		},
-	];
+	const optionsMenuList = optionsMenuListArray(
+		currentPath,
+		bookmarksCountData,
+	).filter((item) => {
+		if (
```

Comment:
We can move all the name like "Inbox" , "Trash" etc .. to constants , what if the name changes in the future ? then if its in constants it will be easy to update